solidity/contracts/bridge/Bridge.sol

```diff
@@ -2120,6 +1636,14 @@ contract Bridge is Ownable, EcdsaWalletOwner {
         return (targetWalletsHash, outputsTotalValue);
     }
 
+    /// @return bank Address of the Bank the Bridge belongs to.
+    /// @return relay Address of the Bitcoin relay providing the current Bitcoin
+    ///         network difficulty.
+    function getContracts() external view returns (Bank bank, IRelay relay) {
```

Comment:
Non-blocking but maybe `getPeerContracts` or something similar? `getContracts` looks very generic IMHO.

Reply:
Yeah, let's revisit in the final cleanup PR. Maybe we will just expose them in separate functions, we have a plenty of space in Bridge now.